openapi_python_client/templates/errors.py.jinja

```diff
@@ -2,6 +2,11 @@
 
 class UnexpectedStatus(Exception):
     """ Raised by api functions when the response status an undocumented status and Client.raise_on_unexpected_status is True """
-    ...
+
+    def __init__(self, status_code: int, content: bytes):
+        self.status_code = status_code
+        self.content = content
+
+        super().__init__(f"Unexpected status: {status_code}")
```

Comment:
```suggestion
        super().__init__(f"Unexpected status code: {status_code}")
```

In case anyone was string-matching, this should make a not a breaking change

Reply:
fixed

Reply:
Confirmed, potentially breaking change reverted.